### PR TITLE
[#336] Removed Hijack

### PR DESCRIPTION
**Pull request by ErhanCitil**

### CODE REVIEW

**Comment by ErhanCitil** [comment]
@alextreme deze kan ook nagekeken worden

File: src/objects/accounts/admin.py [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
 
-from hijack_admin.admin import HijackUserAdminMixin
-
 from .models import User
 
 
 @admin.register(User)
-class _UserAdmin(UserAdmin, HijackUserAdminMixin):
-    list_display = UserAdmin.list_display + ("hijack_field",)
+class UserAdmin(UserAdmin):
```
Comment (alextreme) [inline src/objects/accounts/admin.py:8]: dit kan dan weg volgens mij

Comment (ErhanCitil) [reply src/objects/accounts/admin.py:8]: > dit kan dan weg volgens mij

Ik kan de `@admin.register(User)` niet gebruiken zonder de UserAdmin class

**Review by alextreme — APPROVED** [review]
Fine by me, but I suspect that @annashamray would like to see the hijack-removal and the blacking of the migrations in two separate commits

File: requirements/base.in [hunk 2]
```diff
@@ -36,3 +34,4 @@ elastic-apm  # Elastic APM integration
 # Common ground libraries
 vng_api_common[markdown_docs]>=1.6.4
 zgw-consumers  # external api auths
+markdown==3.4.3
```
Comment (annashamray) [inline requirements/base.in:37]: Please remove pinned version from base.in, we pin versions in base.txt
And why is this package needed?

**Review by annashamray — CHANGES_REQUESTED** [review]
@alextreme I think like this PR and https://github.com/maykinmedia/objects-api/pull/341 contradict each other. In PR #341 hijack is even updated 

I think one of them should be closed

**Pull request closed without merging**